moco/src/main/java/com/cognifide/aem/stubs/moco/MocoStubs.java

```diff
@@ -102,7 +108,9 @@ private void stop() {
   @Override
   public void runScript(Resource resource) {
     final StubScript script = new StubScript(resource, manager, this);
-
+    final JcrResourceReaderFactory jcrResourceReaderFactory = new JcrResourceReaderFactory(resolverAccessor);
```

Comment:
What is a purpose of MethodClosure here?

Reply:
The reason is so that you can use the same syntax as Moco, just method names instead of object.methodName. The methodclosure exposes the object’s method as global function in the groovyscripts instead of resourceFactory.jcr